docs/tutorials/cr_ham_tomo_example.ipynb

```diff
@@ -0,0 +1,304 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# CR Hamiltonian Tomography Demo\n",
+    "\n",
+    "This is a very basic implemention of a CR Hamiltonian tomohraphy experiment.\n",
```

Comment:
A bit more information on what this does is needed here. References would also be good. Perhaps also showing one or two of the circuits that will be run will also help.